client/app/dbaas/cloud-db/advanced-parameter/cloud-db-advanced-parameter-edit.less

```diff
@@ -0,0 +1,14 @@
+.cloud-db-parameters {
```

Comment:
See if in the future we can handle it in a standard way or not.
(Specific or not that's the question)